File: sdk/examples/typescript/quickstart.mjs
import { QtanglClient, newIdempotencyKey, VERSION } from "@qtangl/sdk";

const client = new QtanglClient({
  baseUrl: process.env.QTANGL_API_BASE ?? "https://api.qtangl.com",
  apiKey: process.env.QTANGL_API_KEY ?? "your-api-key",
});

console.log("sdk version", VERSION);

const scan = await client.scanFixture({ idempotencyKey: newIdempotencyKey() });
console.log("scanId", scan.scanId);

const verify = await client.verifyScan(String(scan.scanId));
console.log("valid", (verify.verification as { valid?: boolean } | undefined)?.valid);

const settings = await client.monitor.getSettings();
console.log("settings", settings.settings);

File: sdk/examples/python/quickstart.py
#!/usr/bin/env python3
"""Minimal Qtangl Python SDK quickstart."""

from qtangl import QtanglClient, new_idempotency_key

client = QtanglClient(
    base_url="https://api.qtangl.com",
    api_key="your-api-key",
)

scan = client.scan_fixture(idempotency_key=new_idempotency_key())
print("scanId", scan["scanId"])

verify = client.verify_scan(str(scan["scanId"]))
print("valid", verify.get("verification", {}).get("valid"))

print("schedules", client.list_schedules().get("schedules", []))
print("settings", client.monitor.get_settings().get("settings", {}))
